test/unit/spec/EnrollChoices_spec.js

```diff
@@ -578,6 +603,116 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
         });
       });
     });
+
+    Expect.describe('Multiple Enrollments', function () {
+      Expect.describe('U2F', function () {
+        itp('does not display cardinality text for optional u2f', function () {
+          return setup(resAllFactors).then(function (test) {
+            expect(test.form.factorButtonText('U2F')).toBe('Setup');
+            expect(test.form.factorCardinalityText('U2F')).toBe('');
+          });
+        });
+        itp('Displays correct button and cardinality text when enrolled=1 optional=2', function () {
```

Comment:
nit: `displays` in lower case (it is our common for `itp` descriptions) ->  same comment in all `itp` blocks below